packages/eventsourcing-store-postgres/src/bridge.integration.test.ts

```diff
@@ -0,0 +1,159 @@
+/* eslint-disable effect/no-intermediate-effect-variables -- Integration tests use intermediate variables for test clarity and debugging */
```

Comment:
Notice the other tests do not require all of these. Improve the test code so they are not required. Test code is as important as production code and quality must be maintained.